pkg/registry/core/service/ipallocator/allocator.go

```diff
@@ -187,7 +189,7 @@ func (r *Range) Release(ip net.IP) error {
 // ForEach calls the provided function for each allocated IP.
 func (r *Range) ForEach(fn func(net.IP)) {
 	r.alloc.ForEach(func(offset int) {
-		ip, _ := GetIndexedIP(r.net, offset+1) // +1 because Range doesn't store IP 0
```

Comment:
it also fixed the bug you fixed in the utils repo, right?

Reply:
Yes, after that PR gets merged, we can fix the bug by bumping the `utils` version. 😄

Reply:
should we hold this PR until https://github.com/kubernetes/utils/pull/145 merges then?
Also, don't we have to update the vendor folder with the new code too?

Reply:
ahh ok, I got it, we bump `utils` in a follow-up PR so it automatically gets the fix
scratch las comment